Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -416,6 +416,10 @@ void kekulizeMol(ROMol &mol, bool clearAromaticFlags = false) {
   auto &wmol = static_cast<RWMol &>(mol);
   MolOps::Kekulize(wmol, clearAromaticFlags);
 }
+void kekulizeMolIfPossible(ROMol &mol, bool clearAromaticFlags = false) {
```

Comment:
Is this intended for this PR?  I couldn't find it in the react in place code.

Reply:
I added it when I was planning on also replacing transformation code in MolStandardize with these in-place reactions, but that turned out to not be feasible. Since the code is, I think, super useful, I didn't want to delete it.

If you'd prefer I can pull that out and make a separate PR for it... was just too lazy to do it here.